documentation/book/cluster-operator.adoc

```diff
@@ -1,7 +1,7 @@
 == Cluster Operator
 
 The Cluster Operator is in charge of deploying a Kafka cluster alongside a Zookeeper ensemble.
-As part of the Kafka cluster, it can also deploy the topic operator which provides operator-style topic management via ConfigMaps.
+As part of the Kafka cluster, it can also deploy the topic operator which provides operator-style topic management via `KafkaTopic` custom resources.
```

Comment:
I'm always capital first letters for TO, so Topic Operator (as it happens for the Cluster Operator above). Do we want to write in that way or we have to change consistently across the doc? I see that Topic Operator is used in following files as well.

Reply:
I usually also type it with capital letters. But maybe this would be a good think to discuss on todays meeting with docs. I added it to the agenda.